Sources/SwiftTileserverCache/WebServer.swift

```diff
@@ -37,7 +37,7 @@ public class WebServer {
         
         router.get("/", handler: getRoot)
         router.get("/tile/:style/:z/:x/:y/:scale/:format", handler: getTile)
-        router.get("/static/:style/:lat/:lon/:zoom/:width:/:height/:scale:/:format", handler: getStatic)
+        router.get("/static/:style/:lat/:lon/:zoom/:orientation:/:tilt:/:width:/:height/:scale:/:format", handler: getStatic)
```

Comment:
Instead of making these 2 required make them optional so it still works with old urls

```SWIFT
router.get("/static/:style/:lat/:lon/:zoom/:width:/:height/:scale:/:format", handler: getStatic)
router.get("/static/:style/:lat/:lon/:zoom/:orientation:/:tilt:/:width:/:height/:scale:/:format", handler: getStatic)
```